isisd/isis_lsp.c

```diff
@@ -532,6 +532,9 @@ static void lsp_update_data(struct isis_lsp *lsp, struct isis_lsp_hdr *hdr,
 				: IS_LEVEL_1);
 	}
 
+	_lsp_regenerate_schedule(area, lsp->level, 0, false, __func__, __FILE__,
+				 __LINE__);
+
```

Comment:
You should not use the `_lsp_regenerate_schedule()` function but instead the `lsp_regenerate_schedule()` macro. Please have a look to existing call to this function.